src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        enum ColorBase{
+            BLUE,
+            YELLOW,
+            GREEN,
+            RED;
```

Comment:
The enum `ColorBase` should be declared outside of the method, preferably at the class level. Declaring it inside the method is unconventional and may lead to issues if you need to use the enum elsewhere in your code.